src/test/regress/expected/citus_locks.out

```diff
@@ -0,0 +1,49 @@
+CREATE SCHEMA citus_locks;
+SET search_path TO citus_locks;
+SET citus.next_shard_id TO 1000;
+-- store the current gpid in a variable
+SELECT citus_backend_gpid() \gset
+CREATE TABLE dist_locked_table(id int, data text);
+SELECT create_distributed_table('dist_locked_table', 'id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+BEGIN;
+-- Alter a distributed table so that we get some locks
+ALTER TABLE dist_locked_table ADD COLUMN new_data_column text;
+-- list the locks on relations for current distributed transaction
+SELECT relation_name, nodeid, mode, granted
+FROM citus_locks
+WHERE global_pid = :citus_backend_gpid AND locktype = 'relation' AND relation_name LIKE '%dist_locked_table%'
+ORDER BY 1, 2, 3, 4;
+           relation_name            | nodeid  |        mode         | granted
```

Comment:
I got null nodename and nodeport values for some of the records. My guess is that we have a secondary node with a different `nodecluster` parameter, and `citus_nodename_for_nodeid` and `citus_nodeport_for_nodeid` returns null for all secondary nodes.

Do you think that we should update that in a separate PR?

Reply:
Nevermind, I saw no null values when I ran `multi_1_schedule` instead of `minimal_schedule`. I guess there is not much harm if we return null values for those nodes.